documentation/staging/content/faq/external-clients.md

```diff
@@ -162,6 +244,37 @@ And, here is a snippet of offline WLST code that corresponds to the above `confi
   set('ClientCertificateEnforced', false)
 ```
 
+And, here is a snippet of WDT model YAML configuration that corresponds to the above `snippets`:
```

Comment:
And, here -> Here (you don't need the "And" in this or the above sentence)
YAML  -> YAML file

Reply:
fixed